src/components/Book.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import propTypes from 'prop-types';
+
+const Book = ({ title, author }) => (
+  <div className="bookCard">
+    <p className="title">
+      {title}
+    </p>
+    <p className="author">
+      {author}
+    </p>
+  </div>
+);
```

Comment:
## Please make sure it has a button for removing books:
- The requirements say that the App has a button for removing books and the button wouldn't remove the books yet, this is just a placeholder for now